action.yml

```diff
@@ -12,5 +12,5 @@ inputs:
     description: Export the secrets as environment variables
     default: true
 runs:
-  using: 'node16'
-  main: 'dist/index.js'
+  using: "node16"
+  main: "dist/index.js"
```

Comment:
Just wondering if there's any preference/good practice here as to which one is better suited 🤔

Reply:
I am neutral as long as it's consistent and enforced, which is what appears to be happening

Reply:
Changing the strings from single quotes to double quotes was performed automatically by the 1password/front-end-style Prettier config, so I'll mark this conversation as resolved.